### PR TITLE
Non charged, polar amino acids

### PR DESCRIPTION
**Pull request by SamCox822**

### CODE REVIEW

File: data/aa_polarity/aa_polarity.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: aa_polarity
+language: python
+prompt: aa_polarity.txt
+label: aa_polarity.py
+context: python
```
Comment (navneeth3005) [inline data/aa_polarity/aa_polarity.yml:5]: Will probably need a different context.

File: data/aa_polarity/aa_polarity.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+sample_sequence = "ACRZSSDP"
```
Comment (navneeth3005) [inline data/aa_polarity/aa_polarity.py:2]: Import appropriate packages.

**Review by navneeth3005 — COMMENTED** [review]
The context for this function will depend on how you would actually process the example in aa_polarity.py.

File: data/aa_polarity/aa_polarity.py (hunk 2 above)
Comment (SamCox822) [reply data/aa_polarity/aa_polarity.py:2]: @navneeth3005 Do I need to import packages if my evaluation is a specific example like this?

**Review by navneeth3005 — APPROVED** [review]

**Review by geemi725 — APPROVED** [review]

**Pull request merged**